.github/scripts/otp-compliance.es

```diff
@@ -1605,6 +1614,41 @@ test_copyright_not_empty(#{~"packages" := Packages}) ->
     true = lists:all(fun (#{~"copyrightText" := Copyright}) -> Copyright =/= ~"" end, Packages),
     ok.
 
+test_copyright_format(#{~"packages" := Packages, ~"files" := Files}) ->
```

Comment:
If we are to requite a more strict copyright statement, that should be reflected in the docs: https://github.com/erlang/otp/blob/master/HOWTO/LICENSE-HEADERS.md#copyright-statement.